### PR TITLE
Fix for adversarial visualizations (and fixed num of adversarial examples to be crafted in print statement)

### PR DESCRIPTION
**Pull request by rms5643**
wrote two conditions to actually check the viz_enabled flag as well as adjusted the first print statement that shows how many adversarial examples will be crafted to the correct value (we don't create an adversarial example for the source class, so just had to subtract 1)

### CODE REVIEW

**Comment by goodfeli** [comment]
I'll leave this for someone who uses JSMA to review

File: tutorials/mnist_tutorial_jsma.py [hunk 1]
```diff
@@ -190,7 +191,8 @@ def main(argv=None):
     sess.close()
 
     # Finally, block & display a grid of all the adversarial examples
-    _ = grid_visual(grid_viz_data)
+    if viz_enabled:
```
Comment (npapernot) [inline tutorials/mnist_tutorial_jsma.py]: This looks like a typo, shouldn't it be `FLAGS.viz_enabled`?

Comment (npapernot) [inline tutorials/mnist_tutorial_jsma.py]: @rms5643

**Comment by npapernot** [comment]
LGTM

**Pull request merged**